src/components/DuosHeader.js

```diff
@@ -130,107 +260,153 @@ class DuosHeader extends Component {
+        //I'll look into rewriting the large navbar on a later PR
+        h(Hidden, {lgUp: true}, [
+          div({style: {display: 'flex', flexDirection: 'row', justifyContent: 'space-between'}}, [
+            img({
+              style: duosLogoImage, src: '/images/duos_logo.svg',
+              alt: 'DUOS Logo',
+              onClick: (e) => this.goToLink('/home')
+            }),
+            h(IconButton, {size: 'small', onClick: (e) => this.toggleDrawer(true)}, [
+              h(MenuIcon, {
+                style: {color: 'white', fontSize: '6rem', flex: 1},
+                anchor: 'right',
+              }, [])
+            ]),
+            h(Drawer, {
+              anchor: 'right',
+              open: this.state.openDrawer,
+              PaperProps: {
+                className: classes.drawerPaper
+              },
+              style: {
+                // I have to give this a ridiculous z-index value to keep the google sign in widget or register link from showing up on top
+              },
+              onClose: (e) => this.toggleDrawer(false)
+            }, [
+              h(List, {}, [
+                //NOTE: create user component to show logged in status (as well as dropdown options)
+                h(BasicListItem, {isRendered: isLogged, applyPointer, targetLink: '/profile', label: 'Your Profile', goToLink: this.goToLink}),
+                h(BasicListItem, {isRendered: isAdmin, applyPointer, targetLink: '/admin_console', label: 'Admin Console', goToLink: this.goToLink}),
+                h(BasicListItem, {isRendered: isChairPerson, applyPointer, targetLink: this.state.dacChairPath, label: 'DAC Chair Console', goToLink: this.goToLink}),
+                h(BasicListItem, {isRendered: isMember, applyPointer, targetLink: '/member_console', label: 'DAC Member Console', goToLink: this.goToLink}),
+                h(BasicListItem, {isRendered: isResearcher, applyPointer, targetLink: '/researcher_console', label: 'Researcher Console', goToLink: this.goToLink}),
+                h(BasicListItem, {isRendered: isResearcher, applyPointer, taretLink: '/data_ownder_console', label: 'Data Owner Console', goToLink: this.goToLink}),
+                h(BasicListItem, {isRendered: isResearcher, applyPointer, targetLink: '/dar_application', label: 'Request Application', goToLink: this.goToLink}),
+                h(DropdownComponent, {isRendered: isAdmin, label: 'Statistics', goToLink: this.goToLink, onMouseEnter: applyPointer, dropdownLinks: dropdownLinks.statistics, classes}),
+                h(BasicListItem, {isRendered: isLogged, applyPointer, targetLink: '/dataset_catalog', label: 'Dataset Catalog', goToLink: this.goToLink}),
+                h(BasicListItem, {isRendered: !isLogged, applyPointer, targetLink: '/home_about', label: 'About', goToLink: this.goToLink}),
+                h(BasicListItem, {isRendered: !isLogged, applyPointer, targetLink: '/FAQs', label: 'FAQs', goToLink: this.goToLink}),
```

Comment:
We show FAQs now in both logged in and non-logged in state. We should keep that visible in both versions

Reply:
```
ul({ isRendered: !isLogged, className: 'navbar-public' }, [
    li({}, [
      h(Link, { id: 'link_about', className: 'navbar-duos-link', to: '/home_about' }, [
        div({ className: 'navbar-duos-icon-about', style: navbarDuosIcon }),
        span({ style: navbarDuosText }, ['About'])
      ])
    ]),
    li({}, [
      h(Link, { id: 'link_help', className: 'navbar-duos-link', to: '/FAQs' }, [
        div({ className: 'navbar-duos-icon-help', style: navbarDuosIcon }),
        span({ style: navbarDuosText }, ['FAQs'])
      ])
    ]),
    contactUsButton, supportrequestModal
  ])
])
```

This is still present, which means it's a conditional duplicate, so I might as well remove it.

Reply:
Nevermind, didn't realize the current navbar maintains ul tags for logged in/logged out states